<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>458</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sitka Text</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TERMOMETRO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/Recursos/modo.png</normaloff>:/Recursos/modo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(180, 212, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>171</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 240, 255);</string>
    </property>
    <property name="title">
     <string>Centigrados</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QDial" name="intCent">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(224, 229, 255);</string>
     </property>
     <property name="minimum">
      <number>-273</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="outCent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);</string>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>181</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 240, 255);</string>
    </property>
    <property name="title">
     <string>Fahrenheit</string>
    </property>
    <widget class="QLCDNumber" name="outFahr">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 127);</string>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="intValue" stdset="0">
      <number>32</number>
     </property>
    </widget>
    <widget class="QDial" name="intFahr">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>91</width>
       <height>111</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(224, 229, 255);</string>
     </property>
     <property name="minimum">
      <number>-460</number>
     </property>
     <property name="maximum">
      <number>210</number>
     </property>
     <property name="value">
      <number>32</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>171</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 240, 255);</string>
    </property>
    <property name="title">
     <string>Kelvin</string>
    </property>
    <widget class="QLCDNumber" name="outKel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 255, 202);
</string>
     </property>
     <property name="intValue" stdset="0">
      <number>273</number>
     </property>
    </widget>
    <widget class="QDial" name="intKel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(224, 229, 255);</string>
     </property>
     <property name="maximum">
      <number>372</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>intCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
